specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/blobAuditing.json

```diff
@@ -124,6 +124,51 @@
           }
         }
       }
+    },
```

Comment:
The PR title mentions two List APIs. Is one missing?

Reply:
No, one API works on a database and is present in above file. The other one works on a server, and is placed under 2017-03-01-preview/blobAuditing.json.